src/OpenSage.Game/Diagnostics/GameLoopView.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ImGuiNET;
+
+namespace OpenSage.Diagnostics
+{
+    internal class GameLoopView : DiagnosticView
```

Comment:
Please could you add “sealed” here? There’s supposed to be a potential speed up when the JIT knows a method can’t be overridden anymore.